src/transaction_utils.cpp

```diff
@@ -432,6 +433,17 @@ namespace sdk {
             asset_id_from_json(net_params, addressee));
     }
 
+    void set_tx_output_value(const network_parameters& net_params, wally_tx_ptr& tx, uint32_t index,
+        const std::string& asset_tag, amount::value_type satoshi)
+    {
+        const bool is_liquid = net_params.is_liquid();
+        if (is_liquid) {
```

Comment:
if you aren't re-using the bool (as here) might as well just use `net_params.is_liquid()` in the `if` since its self documenting.

Reply:
Changed thanks